src/Centreon/Domain/Contact/Contact.php

```diff
@@ -602,4 +607,23 @@ public function setUseDeprecatedPages(bool $useDeprecatedPages)
 
         return $this;
     }
+
+    /**
+     * @return bool
+     */
+    public function isOneClickExportEnabled(): bool
+    {
+        return $this->isOneClickExportEnabled;
+    }
+
+    /**
+     * @param bool $isOneClickExportEnabled
+     * @return Contact
+     */
+    public function setOneClickExportEnabled(bool $isOneClickExportEnabled): Contact
```

Comment:
```suggestion
    public function setOneClickExportEnabled(bool $isOneClickExportEnabled): self
```

to be consistent with other setters.